media/CircleEditor/view-sidebar.js

```diff
@@ -538,37 +603,261 @@ class NodeAttributeView {
+                            this._element.appendChild(typeLine);
+                        } else {
+                            typeLine.innerHTML = 'type: ' +
+                                '<code><b>' + this._attribute.type + '</b></code>';
+                            this._element.appendChild(typeLine);
+                        }
+                    } else {
+                        typeLine.innerHTML = 'type: ';
+                        this._select = this._host.document.createElement('select');
+                        this._select.className = 'sidebar-view-item-value-line-type-select';
+                        for (const type of customType) {
+                            const option = this._host.document.createElement('option');
+                            option.setAttribute('value', type);
+                            option.innerText = type.toLowerCase();
+                            if (type.toLowerCase() === this._attribute.type) {
+                                option.setAttribute('selected', 'selected');
+                            }
+                            this._select.appendChild(option);
+                        }
+                        typeLine.appendChild(this._select);
+                        this._element.appendChild(typeLine);
+                    }
 
-    /**
-     * TODO Implement makeEditCustomObject()
-     * Object to send when modifying attributes
-     */
+                    const description = this._attribute.description;
+                    if (description) {
+                        const descriptionLine = this._host.document.createElement('div');
+                        descriptionLine.className = 'sidebar-view-item-value-line-border';
+                        descriptionLine.innerHTML = description;
+                        this._element.appendChild(descriptionLine);
+                    }
+                }
+
+                if (this._attribute.type === 'tensor' && value) {
+                    const state = value.state;
+                    const valueLine = this._host.document.createElement('div');
+                    valueLine.className = 'sidebar-view-item-value-line-border';
+                    const contentLine = this._host.document.createElement('pre');
+                    contentLine.innerHTML = state || value.toString();
+                    valueLine.appendChild(contentLine);
+                    this._element.appendChild(valueLine);
+                }
+        }
+    }
+
+    save() {
+        if (this._attribute._type === 'int32') {
+            if (this._line.value - 0 > 2147483648) {
+                // TODO Enable posting message
+                // vscode.postMessage({
+                //     command : 'alert',
+                //     text: 'Can\'t exceed 2,147,483,648'
+                // });
+                return;
+            }
+        }
+
+        // TODO It will be implemented when the attribute type of the CustomOperator can be
```

Comment:
Isn't it available to enable it?

Reply:
An error occurs because the function that receives the type of the custom operator's type is not implemented in the DS side yet.

Reply:
But you disabled posting a message and leave it as TODO....?